platform/lib/routers/runtimeLog.js

```diff
@@ -15,33 +15,45 @@
  */
 
 'use strict';
+
 const express = require('express');
-const LogFormatter = require('../runtime-log/formatter');
+const LogFormatter = require('../runtime-log/HtmlFormatter.js');
 
-const Templates = require('@lib/templates/');
+const Templates = require('../templates/');
```

Comment:
Just curious: is there a reason you've been moving away from module-register syntax (`@lib/`)?

Reply:
Jest doesn't support it....